src/Apps/Order/Components/ShippingAndPaymentDetails.tsx

```diff
@@ -1,38 +1,117 @@
 import React from "react"
 
 import { Serif, space } from "@artsy/palette"
+import { ShippingAndPaymentDetails_order } from "__generated__/ShippingAndPaymentDetails_order.graphql"
+import { createFragmentContainer, graphql } from "react-relay"
 import { StackableBorderBox } from "Styleguide/Elements/Box"
-import { Flex } from "Styleguide/Elements/Flex"
-import {
-  CreditCardIcon,
-  CreditCardType,
-} from "Styleguide/Elements/icons/CreditCardIcon"
+import { Flex, FlexProps } from "Styleguide/Elements/Flex"
+import { CreditCardIcon } from "Styleguide/Elements/icons/CreditCardIcon"
 
-interface ShippingAndPaymentDetailsProps {
-  address: string
-  creditCardLast4digits: string
-  creditCardtype: CreditCardType
-  creditCardExpiry: string
+interface ShippingAndPaymentDetailsProps extends FlexProps {
+  order: ShippingAndPaymentDetails_order
+}
+
+const renderAddress = ({
```

Comment:
This can come in a separate PR, but it would be nice to add some tests for this screen (especially with the logic around rendering the address/conditionally showing the pickup option).